public_html/wp-content/plugins/pattern-creator/pattern-creator.php

```diff
@@ -95,12 +95,21 @@ function pattern_creator_init() {
 	);
 	wp_set_script_translations( 'wp-pattern-creator', 'wporg-patterns' );
 
+	wp_add_inline_script(
+		'wp-pattern-creator',
+		sprintf(
+			'var wporgLocale = JSON.parse( decodeURIComponent( \'%s\' ) );',
+			rawurlencode( wp_json_encode( get_locale() ) )
+		),
+		'before'
+	);
+
 	wp_add_inline_script(
 		'wp-pattern-creator',
 		sprintf(
 			'var wporgBlockPattern = JSON.parse( decodeURIComponent( \'%s\' ) );',
 			rawurlencode( wp_json_encode( array(
-				'siteUrl'       => esc_url( home_url() ),
+				'siteUrl' => esc_url( home_url() ),
```

Comment:
Just noting that this change appears to be a formatting update. Not sure if you intended to commit it.

Reply:
Yep, since I was already working in the file I fixed this — at some point we must've removed other properties but didn't fix the alignment.